conduct.md

```diff
@@ -7,16 +7,282 @@
+    towards the project.
+    * Make decisions with the best interest of the team as a whole upon proper
+    discussion.
+* **Unacceptable Behavior (U)**
+  * Team Members shall NOT:
+    * Bring about personal interests that will bring about conflicts with
+    those of the team as a whole.
+    * Move forward on a conflict that has not yet been discussed or resolved
+    to prevent even further conflicts to arise.
+  * Team Leads shall NOT:
+    * Disregard any conflict that has been brought to their attention.
+    * Use their position/ influence to make any type of decision that is not
+    beneficial to the team as a whole.
+
+## Infractions
+
+An ***infraction*** shall hereby be defined as a penalty given as a result of a
+violation of the above outlined Conduct Guide by any individual who has agreed
+to adhere to the Conduct Guide via digital signature below. The infractions for
+any individual shall be recorded and managed by the instructor.
+
+A *violation* of the Conduct Guide is defined as any behavior that results in receiving
+a "U" (Unacceptable) for any of the above outlined (sub)sections.
+
+Infractions shall be imposed at three different levels of severity:
+
+* **Level I**: A ***Level I infraction*** is defined as a formally documented
+ violation of the above outlined Conduct Guide. A Level I infraction may be
+ suggested to the instructor by a team or project leader for any given project.
+ All Level I infractions will be reset weekly.
+* **Level II**: A ***Level II infraction*** will be given to an individual who
+ has accumulated three Level I infractions. Upon obtaining a Level II infraction,
+ the individual will no longer be eligible to receive a check for the week. All
+ Level II infractions will be reset weekly.
+* **Level III**: A ***Level III infraction*** will be given to an individual who
+ has accumulated five Level I infractions. Upon obtaining a Level III infraction,
+ the individual will be docked 10% of their grade for the current project. All
+ Level III infractions will be reset after the completion of the current project.
+
+If an individual commits multiple violations of the Conduct Guide, a team or
+project leader may suggest a proportional amount of Level I infractions for the
+violations to the instructor, provided there is proper citation to the Conduct
+Guide for the violations. While a team or project leader may suggest any number
+of infractions, an individual *should* not incur more than **three** infractions
+for a single instance of violations. The number of Level I infractions imposed
+upon the individual shall ultimately be determined by the instructor.
+
+An individual who has received any number of infractions may appeal said
+infraction(s) directly to the instructor of the course. Any decision received
+from the instructor for being given the infraction(s) will be final.
+
 ## Names of Individuals who Agree to Adhere to the Guide
```

Comment:
For ease of use, I would suggest removing this section and instead using Slack or a GitHub issue to collect "virtual signatures".